fpm_template.go

```diff
@@ -136,7 +139,8 @@ arch: {{ .Arch }}
 platform: linux
 release: 1
 version: {{ .Version }}
-maintainer: {{ .Maintainer }}
+maintainer: {{ .Maintainer}}
+vendor: {{ .Vendor }}
```

Comment:
Q: If these are now effectively entirely optional, what's the behavior in this template with empty values?

In my head I'm thinking RPM distinguishes between undefined and empty-string. But our template isn't doing that, it's setting all these tags to `""`.

I think we should keep the behavior we are already doing (what you've done here). But it does make me wonder if we should instead be handling this better by _not_ be defining the tag when it's empty.

```
{{ if .Maintainer }}
maintainer: {{.Maintainer}}
{{end}}
```

... Which is annoying and out of scope for this PR.

Reply:
Good point, and agreed, wouldn't change the way things currently behave. I created in our backlog to tackle this separately (RELENG-397)